### DISCUSSION TITLE
Upstream response time mismatch.

### DISCUSSION THREAD

**Discussion by ksgnextuple** [Kong/kong#13300] [Help]
Hi Team

We are running few performance tests on of our apis and we see that the kong_upstream_response time reported on Prometheus is having a large difference as compared to the actual upstream latency.

Kong : 3.6
KIC: 3.1

The same test was run using Kong 2.8 and KIC 2.5 we saw that this difference was minimal which is expected behavior.

Has anyone faced a such an issue?

**Comment by hanshuebner** [comment]
Superseded by https://github.com/Kong/kong/issues/13302
